packages/esm-appointments-app/src/home-appointments/appointments-list.scss

```diff
@@ -145,3 +144,11 @@
 .menuItemLink {
   text-decoration: none;
 }
+
+.seeAllLink {
+  position: relative;
```

Comment:
Instead of positioning relatively is it possible to add it as a component to the footer of the table? I'm worried this approach will break easily

Reply:
Without relative position, it can not be displayed on top of the pagination component, it just appears below it and there is no way to pass any children to the pagination component, so if we are to follow the mock ups, I think relative positioning is the best option.
<img width="1418" alt="Screenshot 2022-09-22 at 14 08 44" src="https://user-images.githubusercontent.com/30952856/191731806-2dd3ac76-2c5d-45b9-86ee-7a534b8a81e1.png">